### PR TITLE
Iris Beta Test v0.91

### PR DESCRIPTION
**Pull request by luckyshotpictures**
## About the PR

Iris Station, Map rendered soon:tm: after I slept

## Media

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

### CODE REVIEW

File: Resources/Maps/_Umbra/Iris.yml
Comment (TsjipTsjip) [inline Resources/Maps/_Umbra/Iris.yml:1]: Rename the file to `iris.yml`.

File: Resources/Prototypes/_Umbra/Maps/iris-prototype.yml
Comment (TsjipTsjip) [inline Resources/Prototypes/_Umbra/Maps/iris-prototype.yml:1]: Rename the file to `iris.yml`.

File: Resources/Prototypes/_Umbra/Maps/iris-prototype.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
+- type: gameMap
+  id: Iris
+  mapName: 'Iris'
+  mapPath: /Maps/_Umbra/Iris.yml
```
Comment (TsjipTsjip) [inline Resources/Prototypes/_Umbra/Maps/iris-prototype.yml:4]: Should be updated if the map file name was updated already too.

File: Resources/Prototypes/_Umbra/Maps/iris-prototype.yml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        - type: StationNameSetup
+          mapNameTemplate: '{0} Iris {1}'
+          nameGenerator:
+            !type:NanotrasenNameGenerator
+            prefixCreator: '14'
+        - type: StationEmergencyShuttle
+          emergencyShuttlePath: /Maps/Shuttles/emergency_courser.yml
+        - type: StationJobs
+          overflowJobs:
+            - Passenger
```
Comment (TsjipTsjip) [inline Resources/Prototypes/_Umbra/Maps/iris-prototype.yml:20]: This no longer exists, instead, define `Passenger` in the list of available jobs with tresholds `-1` and `-1`.

**Review by TsjipTsjip — CHANGES_REQUESTED** [review]

File: Resources/Maps/_Umbra/Iris.yml
Comment (TsjipTsjip) [inline Resources/Maps/_Umbra/Iris.yml:1]: Rename the file to `iris.yml`, be mindful not to use an uppercase I.

**Review by TsjipTsjip — APPROVED** [review]

**Review by TsjipTsjip — DISMISSED** [review]
I messed up the last one. Treat it as a requested changes.

**Comment by luckyshotpictures** [comment]
Fuck it, I will be remaking Iris to be better™

**Pull request closed without merging**